app/main.py

```diff
@@ -1,25 +1,89 @@
-from abc import ABC
+from __future__ import annotations
+from abc import ABC, abstractmethod
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Visitor, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: Visitor, owner: Visitor) -> Visitor:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: Visitor, value: int) -> None:
+        return setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+
+    age = IntegerRange(4, 60)
+    weight = IntegerRange(20, 120)
+    height = IntegerRange(80, 220)
+
```

Comment:
Do not declare class attributes at the `Visitor` class, it should have only `__init__`.